src/client/driver/command-executors/execute-hybrid-fn-command.js

```diff
@@ -0,0 +1,21 @@
+import hammerhead from '../deps/hammerhead';
+
+var Promise = hammerhead.Promise;
+
+export default function executeHybridFnCommand (command) {
+    return new Promise(resolve => {
+        try {
+            /* eslint-disable no-eval */
+            var fn = eval(command.fnCode);
```

Comment:
Nope. We should use in-place eval here to close on lexical context (we will need it lately). Also, I'm not sure if saved `eval` will behave correctly \cc @churkin